packages/flutter/lib/src/cupertino/thumb_painter.dart

```diff
@@ -43,22 +30,16 @@ const List<BoxShadow> _kSliderBoxShadows = <BoxShadow> [
   ),
 ];
 
-/// Paints an iOS-style slider thumb or switch thumb.
+/// Paints an iOS-style slider thumb.
 ///
-/// Used by [CupertinoSwitch] and [CupertinoSlider].
+/// Used by [CupertinoSlider].
 class CupertinoThumbPainter {
   /// Creates an object that paints an iOS-style slider thumb.
   const CupertinoThumbPainter({
     this.color = CupertinoColors.white,
     this.shadows = _kSliderBoxShadows,
   });
 
-  /// Creates an object that paints an iOS-style switch thumb.
-  const CupertinoThumbPainter.switchThumb({
```

Comment:
Should we deprecate this? I wouldn't be surprised if nobody uses it, but it's technically a public API.

Reply:
That's true. It's not used anywhere else in the framework at least, but this would not be the way to deprecate it anyways.